packages/react-components/src/layout/index.tsx

```diff
@@ -28,6 +28,7 @@ interface MainLayoutProps {
 const Main: FC<MainLayoutProps> = memo(({ children, enableCollapse = true, sidebar }) => {
   const { t } = useTranslation('react-components');
   const { init } = useApi();
+  const result = useFetch('https://ipapi.co/json/');
```

Comment:
I would suggest using https://api.myip.com instead since this one is not working correctly.
```suggestion
  const result = useFetch('https://api.myip.com');
```